datalad/distribution/add.py

```diff
@@ -119,8 +119,16 @@ class Add(Interface):
             transport"""),
         recursive=recursion_flag,
         recursion_limit=recursion_limit,
+        ds2super=Parameter(
+            args=("--ds2super", "--datasets-to-super",),
```

Comment:
Fingers can be added to everything. If they are sufficiently tiny, they can even be added to the White House!

I agree to @mih - could be confusing. But then we might want to call it the same for all commands.